e2e/commands_test.go

```diff
@@ -18,7 +18,13 @@ import (
 	"gotest.tools/skip"
 )
 
+// Skipping experimental e2e test on rendering with templates, as it needs now to set
+// the DOCKERAPP_RENDERERS environment variable inside the invocation image,
+// and the only way to do it is to add a new parameter to the bundle.json.
+// This test should be unskipped or removed as soon as the templates are
+// not experimental anymore, or are removed.
```

Comment:
In principal couldn't we solve that now, by having code which adds that parameter hardcoded when building in experimental mode (and requiring matching experimental enablement in the tool and the invoc image base, which I think is reasonable)?

To be clear though, I can for sure see an argument for not putting a lot of effort into experimental mode right now since we aren't really actively using it.

Reply:
To be honest I don't wan't to spend more time than I already have on that. I think we can fill another issue on that subject and work on it after the release.

Reply:
Sure, I was really just saying that the comment is a spurious justification when there is an actual justification..